tensorboard/plugins/graph/tf_graph_common/scene.ts

```diff
@@ -216,23 +216,29 @@ export function panToNode(nodeName: String, svg, zoomG, d3zoom): boolean {
     return end < 0 || start > bound;
   };
   let svgRect = svg.getBoundingClientRect();
-  if (isOutsideOfBounds(pointTL.x, pointBR.x, svgRect.width) ||
-      isOutsideOfBounds(pointTL.y, pointBR.y, svgRect.height)) {
+
+  // Subtract to make sure that the node is not hidden behind the minimap.
+  const horizontalBound = svgRect.left + svgRect.width - 320;
```

Comment:
Is there a way to avoid these magic numbers?

Reply:
I moved the numbers to constants.